cogs/deepfry.py

```diff
@@ -0,0 +1,65 @@
+from io import BytesIO
+import discord
+import requests
+from PIL import Image, ImageOps, ImageEnhance
+from discord.ext import commands
+from cogs.base import BaseCog
+
+FILE_TYPES = [
```

Comment:
move these static variables to attributes of the class Deepfry so we limit their scope